.github/workflows/cla.yml

```diff
@@ -0,0 +1,36 @@
+name: "CLA Assistant"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+jobs:
+  CLAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        # Beta Release
+        uses: cla-assistant/github-action@v2.1.3-beta
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # the below token should have repo scope and must be manually added by you in the repository's secret
+          PERSONAL_ACCESS_TOKEN : ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+        with:
+          path-to-signatures: '.github/legal/signatures/version1/cla.json'
+          path-to-document: '.github/legal/cla-individual.md' 
```

Comment:
I'm pretty sure the documentation said that `path-to-document` needs to be an absolute URL.

Reply:
I can check that, but then I'd have to put the "future path", corresponding to the one we would have after this PR is merged. Right?

Reply:
Yes. So it would be: `https://github.com/OpenIPSL/OpenIPSL/blob/master/.github/legal/cla-individual.md`